README.md

```diff
@@ -88,22 +88,12 @@ is CentOS 8.
 
 This is currently working only if converting with `convert-with-prep`.
 
-To build the image, run:
-
-```
-$ podman build \
-    [--build-arg base_image=centos:8] \
-    [--build-arg package_manager="yum -y"]  \
-    -t dist2src .
-```
-
-The build arguments are optional, the defaults being `centos:8` and `yum -y`.
+To build the image, run `make build`.
 
 To run the conversion:
 
 ```
-podman run --rm -v $PWD:/workdir:z \
-    dist2src dist2src convert-with-prep rpms/<package>:<branch> source-git/<package>:<branch>
+CONTAINER_CMD="dist2src convert rpms/<package>:<branch> source-git/<package>:<branch>" make run
```

Comment:
I'm not sure about this... `make run` uses `dist2src` code mounted from the current directory, which means, this command needs to be run from the directory where `dist2src` code is checked out, which means input (`rpms`) and output (`source-git`) location needs to be in the location where this repo is cloned, there is no way for someone to just pull the image from some registry and use it without checking out the code. Or build the image and then use in some other location then the `dist2src` repo.

Of course there can be an example to show how to use a "development" version of `dist2src` with `make run`, but let's keep the `podman` approach, too, for users who might not be interested in cloning the code.

Reply:
done